cloud/logger/src/Logger.ts

```diff
@@ -0,0 +1,8 @@
+interface Logger {
+  debug(message: string, data?: any, source?: string, tags?: string[]): void;
```

Comment:
disagree, I think we should have one interface. Browserify use case was just an example, source tag is recommended practice outlined in engineering standards linked here https://issues.jboss.org/browse/RAINCATCH-989